jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/DefaultServletTest.java

```diff
@@ -2001,10 +2002,9 @@ public void testDirectFromResourceHttpContent() throws Exception
         assertThat(response.toString(), response.getStatus(), is(HttpStatus.OK_200));
         assertThat(response.getContent(), containsString("<h1>Hello World</h1>"));
 
-        ResourceHttpContentFactory factory = (ResourceHttpContentFactory)context.getServletContext().getAttribute("resourceCache");
-
         /*
         TODO: fix after HttpContent changes.
+        ResourceHttpContentFactory factory = (ResourceHttpContentFactory)context.getServletContext().getAttribute("resourceCache");
```

Comment:
Because I started fixing it in ResourceServlet, but then saw that the commented out section was beyond the scope of this PR.  So leaving DefaultServletTest as it was, but incremental improvement to ResourceServletTest (but more to do).